src/main/java/seedu/address/model/client/insurance/claim/Claim.java

```diff
@@ -43,6 +43,19 @@ public Claim(String claimId, boolean isOpen, int claimAmount) {
         this.claimAmount = claimAmount;
     }
 
+    /**
+     * Constructs a claim object based on claim id only. This constructor is used, usually during deletion of claims.
+     * The {@code isOpen} and {@code claimAmount} attributes are set to default values, but they are not relevant as
+     * claims are deleted solely based on the claim id.
+     *
+     * @param claimId     claim id obtained through official channels.
+     */
+    public Claim(String claimId) {
+        this.claimId = claimId;
+        this.isOpen = true;
+        this.claimAmount = 0;
+    }
```

Comment:
I think instead of this, it would be more beneficial to create a `getClaim()` method using the claim id string. This would also handle cases such as invalid claims more efficiently without checking for specific edge cases. What do you think?

Reply:
Good idea, will implement this

Reply:
Yup, I agree with Rezwan. This way, it is more consistent with our DeleteInsuranceCommand too.